### DISCUSSION TITLE
check if mongoose connected on vercel

### DISCUSSION THREAD

**Discussion by m0x61h0x64i** [Automattic/mongoose#12317] [Q&A]
hi, i deployed my application on vercel but there is no terminal and logs to see if connection to mongoose was successful or not, i tried to use file system to write a file and look for any logs but it seems vercel dose not support fs, so do you know any other ways that i can check if mongoose connected to my mongodb database or not? please help

**Comment by vkarpov15** [comment]
`console.log(mongoose.connection.readyState)` presuming you're using `mongoose.connect()`. Output is a number, [here's how to interpret the output](https://github.com/Automattic/mongoose/blob/71478c4ff64a8afa6007c9d246e4403f701ed48a/lib/connectionstate.js#L16-L20).
